payjoin/Cargo.toml

```diff
@@ -25,7 +25,7 @@ bitcoin = { version = "0.30.0", features = ["base64"] }
 bip21 = "0.3.1"
 chacha20poly1305 = { version = "0.10.1", optional = true }
 log = { version = "0.4.14"}
-ohttp = { version = "0.4.0", optional = true }
+ohttp = { version = "0.5.0", optional = true }
 bhttp = { version = "0.4.0", optional = true }
```

Comment:
ohttp and bhttp are sibling dependencies. they should both be updated to minimize deps.